src/utils/embeds.ts

```diff
@@ -27,3 +29,32 @@ export const sendKickEmbed = async (message: Message, user: User, reason = '', i
   }
   await (client.channels.cache.get(NOTIF_CHANNEL_ID) as TextChannel).send(kickEmbed);
 };
+
+/*
+* Send embed for getting an example resume
+*/
+export const getResumeEmbed = (domain: string, term: string, resumes: any): MessageEmbed => {
+  try {
+    // get link to resume (just first one in the "list" for now)
+    const resumeLink = resumes[domain][term][0].url;
+    const domainName = availableDomains[domain];
+
+    return new MessageEmbed()
```

Comment:
I don't think this is the best place for the embed
Can you peek as any ``suggestionX.ts`` and see how embeds are done? Later on, we will have many ``resumeX.ts`` as well.

Reply:
Maybe use the suggestions commands / structure to feel how resumes will be done.